src/main/java/frc/robot/commands/SwerveJoystickCommand.java

```diff
@@ -29,19 +26,22 @@ public SwerveJoystickCommand(SwerveDriveSubsystem drivetrain, XboxController con
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
-    
+
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
     // Just think really hard about why these values are negated and flip-flopped.
     // Maybe use a whiteboard or piece of paper.
```

Comment:
should we write why they are negated and flipflopped in a javadoc so future people don't have to think really hard?

Reply:
Lol that might be a good idea.  Or, in ur curriculum materials folder in the google drive, mention it?

Reply:
yeah that could be part of the swerve explanation, but it shouldnt ONLY be there, this is javadoc material. doesn't need to be added now, but eventually we'll need to do a full cleanup